test/util/bspNet/docker.ts

```diff
@@ -33,3 +37,98 @@ export const showContainers = () => {
     console.log("Error displaying docker containers");
   }
 };
+
+export const addBspContainer = async (options?: {
```

Comment:
Would be extremely useful to be able to pass additional command line arguments to the BSP to add. For example, we're gonna have to do integration tests for the RocksDB implementation of our storage layer. That requires having a BSP that was spawned with `--storage-layer rocks-db`

Reply:
Trailing commas error is because you are running old biome - the new key was introduced in biome 1.8.2 (to replace the old key trailingComma).

Updating your `pnpm` pkgs should fix this for you.

Reply:
Added additional Args parameter to helper as an option